projects/hslayers/src/components/utils/utils.service.ts

```diff
@@ -368,6 +374,18 @@ export class HsUtilsService {
       objectToCheck && {}.toString.call(objectToCheck) === '[object Object]'
     );
   }
+  createQueue(useCase: string, customConcurrency?: number): void {
```

Comment:
Maybe ensureQueue would be better name since it creates only if it doesn't exist already?